smll/FaceDetector.cpp

```diff
@@ -145,46 +145,10 @@ namespace smll {
 		char* cropData = detect.data +
 			(detect.getStride() * cropInfo.offsetY) +
 			(detect.getNumElems() * cropInfo.offsetX);
-		switch (detect.type) {
-		case IMAGETYPE_GRAY:
-		{
-			cv::Mat gray(cropInfo.height, cropInfo.width, CV_8UC1, cropData, m_detect.getStride());
-			currentImage = gray.clone();
-			break;
-		}
-		case IMAGETYPE_RGB:
```

Comment:
The reason for creating multiple cases is to support plugin for different image formats. Yes, we are currently receiving only GrayScale but in case if our plugin receives some other format, these switch cases can handle them in future.